game.py

```diff
@@ -821,10 +827,15 @@ def save_game(self, auto: bool = False):
         Saves the current game with the current timestamp as the file name.
         """
         # Only maintain 3 autosaves at a time, delete the oldest if we already have 3 before saving the next.
-        if auto and len(autosaves := list(filter(lambda fn: fn.startswith("auto"), os.listdir("saves/")))) == 3:
+        if auto and len(
+                autosaves := list(filter(lambda fn: fn.startswith(AUTOSAVE_PREFIX), os.listdir(SAVES_DIR)))) == 3:
             autosaves.sort()
-            os.remove(f"saves/{autosaves[0]}")
-        save_name = f"saves/{'auto' if auto else ''}save-{datetime.datetime.now().isoformat(timespec='seconds')}.json"
+            os.remove(os.path.join(SAVES_DIR, {autosaves[0]}))
+        save_name = os.path.join \
+            (SAVES_DIR,
```

Comment:
Some funky formatting here, the open bracket needs to be immediately after the `join`

Reply:
Fixed in [d1ad791](https://github.com/ChrisNeedham24/microcosm/pull/70/commits/d1ad79100f613f104d44f75c4c56ee66be9a933c)